automl/cloud-client/src/main/java/com/example/automl/LanguageSentimentAnalysisPredict.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.automl;
+
+// [START automl_language_sentiment_analysis_predict]
+import com.google.cloud.automl.v1.AnnotationPayload;
+import com.google.cloud.automl.v1.ExamplePayload;
+import com.google.cloud.automl.v1.ModelName;
+import com.google.cloud.automl.v1.PredictRequest;
+import com.google.cloud.automl.v1.PredictResponse;
+import com.google.cloud.automl.v1.PredictionServiceClient;
+import com.google.cloud.automl.v1.TextSnippet;
+
+import java.io.IOException;
+
+class LanguageSentimentAnalysisPredict {
+
+  static void predict() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "YOUR_PROJECT_ID";
+    String modelId = "YOUR_MODEL_ID";
+    String content = "text to predict";
+    predict(projectId, modelId, content);
+  }
+
+  // Predict
+  static void predict(String projectId, String modelId, String content) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (PredictionServiceClient client = PredictionServiceClient.create()) {
+      // Get the full path of the model.
+      ModelName name = ModelName.of(projectId, "us-central1", modelId);
+      TextSnippet textSnippet =
+          TextSnippet.newBuilder()
+              .setContent(content)
+              .setMimeType("text/plain") // Types: text/plain, text/html
```

Comment:
This is great, but linking to the documentation where these types are located might be more helpful if they change. Up to you.

Reply:
I'll create a GitHub issue for myself to add this once docs pages become available.

Reply:
https://github.com/GoogleCloudPlatform/java-docs-samples/issues/1730